src/app/facility/facility-details/facility-details.component.ts

```diff
@@ -224,6 +224,7 @@ export class FacilityDetailsComponent implements OnInit, OnDestroy {
     if (this.searchParams['passType']) {
       this.passTypeSelected = this.searchParams['passType'];
       delete this.searchParams['passType'];
+      this.bookingTimeSummary.capacity = null;
```

Comment:
I think this is a bit dangerous without addressing the math on line 296:

![image](https://user-images.githubusercontent.com/51633/182660158-701df668-acac-481b-91fa-fdc42bc86920.png)

Reply:
@mark I added a safety-check for division by null/zero.  It will divide by `Number.MAX_SAFE_INTEGER` and show as zero percent in the UI.

Reply:
I think simpler if we just 
```
this.bookingTimeSummary.capPercent = this.bookingTimeSummary.capacity === null ? 0 : Math.floor((this.bookingTimeSummary.reserved / this.bookingTimeSummary.capacity) * 100);
```
rather than actually doing a division when it's null.

Reply:
I updated it to use a ternary operator, but I reversed it from your code above to also handle the case where facility capacity has been set to zero.